src/components/ModulesItem/ModulesItem.module.css

```diff
@@ -0,0 +1,20 @@
+.module {
+  display: flex;
+  flex-direction: row;
+}
+
+.modulesItem {
+  background-color: rgb(243, 243, 243);
+  display: flex;
+  flex-direction: column;
+  border-radius: 10px;
+  padding: 20px;
+  width: 80vw;
+  margin: 10px auto;
```

Comment:
It is better to remove this property and use `margin` in the rarest cases, it is better to use `padding` instead or `gap` for flexbox.
The problem is that margin pushes elements further away from each other, but unlike gap, it adds space on both sides of all elements. So the layout can be off and all elements can move away from each other.
In your case (since you have everything on flexbox) - it's better to add `gap: 10px; `property to selector `.container` of `ModulesList` component.
This will be more correct and it will be easier to use this component in the future.
Please remove margin everywhere and add a gap to their parent elements

Reply:
This code section has been processed and corrected